tabula/wrapper.py

```diff
@@ -29,6 +29,8 @@
 JAR_NAME = "tabula-{}-jar-with-dependencies.jar".format(TABULA_JAVA_VERSION)
 JAR_DIR = os.path.abspath(os.path.dirname(__file__))
 JAR_PATH = os.path.join(JAR_DIR, JAR_NAME)
+if os.environ.get("TABULA_JAR"):
```

Comment:
I think this could be better:

```suggestion
if os.environ.get("TABULA_JAR"):
    JAR_PATH = os.path.abspath(os.environ.get("TABULA_JAR"))
else:
    JAR_NAME = "tabula-{}-jar-with-dependencies.jar".format(TABULA_JAVA_VERSION)
    JAR_DIR = os.path.abspath(os.path.dirname(__file__))
    JAR_PATH = os.path.join(JAR_DIR, JAR_NAME)
```

Remove previous lines avoid to duplication.

Reply:
That works for me!

Reply:
Can you apply the patch or push to fix this?